lib/java/com/google/android/material/chip/ChipDrawable.java

```diff
@@ -1652,6 +1682,10 @@ public void setChipIconResource(@DrawableRes int id) {
     setChipIcon(AppCompatResources.getDrawable(context, id));
   }
 
+  public void setChipIconUseDrawableSize(boolean useDrawableSize) {
```

Comment:
1) The getter is missing
2) please add both getter/setter to Chip.java

Reply:
Added getter to ChipDrawable.java and getter/setter to Chip.java